AWS/windows/windows-2012r2-us-west-2.json

```diff
@@ -0,0 +1,8 @@
+{
+  "_comment": "Build with `packer build -var-file=windows-2012r2-us-west-2.json windows.json`",
+  "aws_region": "us-west-2",
+  "ami_filter": "Windows_Server-2012-R2_RTM-English-64Bit-Base-*",
+  "ami_owner": "801119661308",
+  "ami_name": "saltstack/base/windows/2012r2",
+  "os_version": "2012r2"
```

Comment:
os_version is not needed, `make` will pass it

Reply:
Ahh I was just using packer command, should we keep it there so all ways work?

Reply:
In that case, lets also add os_name, and to all var files?

Reply:
@s0undt3ch this should be done now.